src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/grid.controller.js

```diff
@@ -343,6 +343,37 @@ angular.module("umbraco")
             }
         };
 
+        var shouldApply = function(item, itemType, gridItem) {
+            if (item.applyTo === undefined || item.applyTo === null || item.applyTo === "") {
+                return true;
+            }
```

Comment:
If applyTo is undefined, null or empty string (all ways of saying it is not set to anything useful) .. it should just be shown on all rows/cells.